pkgs/development/interpreters/python/hooks/unittest-check-hook.sh

```diff
@@ -0,0 +1,29 @@
+# Setup hook for unittest.
+echo "Sourcing unittest-check-hook"
+
+unittestCheckPhase() {
+    echo "Executing unittestCheckPhase"
+    runHook preCheck
+
+    eval "@pythonCheckInterpreter@ -m unittest discover $unittestFlags"
```

Comment:
```suggestion
    eval "@pythonCheckInterpreter@ -m unittest discover $unittestFlagsArray"
```

To mimic `pytestFlagsArray` and other similar arrays.

Reply:
I originally used that name, but then dropped it because `pytestCheckHook` was only using it so it could use the variable name `pytestFlags`, I guess I should switch it back if just for consistency like you said? ~~(A  bunch of other hooks don't use the array suffix, but that's a whole other issue...)~~

Reply:
Hrm, let's keep it consistent within Python and have the architecture team hash things out. There's at least also buildFlagsArray and configureFlagsArray out there.

Reply:
Done.